install-boost.sh

```diff
@@ -24,12 +24,12 @@ download_boost_source() {
 }
 
 boost_libs="${boost_libs=filesystem,regex,system}"
-boost_cxxflags='-arch arm64 -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk'
```

Comment:
No, that change with `toolset` doesn't matter.
The original script worked okay for me locally without `-isysroot`.
It turns out setting sdk path is a fix for Xcode 12 beta in the early Travis CI osx universal image.